src/EditorFeatures/CSharpTest/CodeActions/ReplacePropertyWithMethods/ReplacePropertyWithMethodsTests.cs

```diff
@@ -1325,6 +1325,97 @@ private int GetProp()
 }", ignoreTrivia: false, options: PreferExpressionBodiedMethods);
         }
 
+        [WorkItem(18234, "https://github.com/dotnet/roslyn/issues/18234")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplacePropertyWithMethods)]
+        public async Task TestDocumentationComment1()
+        {
+            await TestInRegularAndScriptAsync(
+@"internal interface ILanguageServiceHost
+{
+    /// <summary>
+    ///     Gets the active workspace project context that provides access to the language service for the active configured project.
+    /// </summary>
+    /// <value>
+    ///     An value that provides access to the language service for the active configured project.
```

Comment:
As a frequent reviewer of your tests, I don't find them self-descriptive.  Having the name call out the interesting differentiating thing that is the reason this particular test exists is valuable.  Otherwise, reviewers are left mentally diffing the test to all of the other ones to try to understand it's purpose.